packages/components/src/components/contract-card/contract-card-items/accumulator-card-body.jsx

```diff
@@ -0,0 +1,128 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { isCryptocurrency, getLimitOrderAmount, isValidToSell } from '@deriv/shared';
+import ContractCardItem from './contract-card-item.jsx';
+import ToggleCardDialog from './toggle-card-dialog.jsx';
+import Icon from '../../icon';
+import MobileWrapper from '../../mobile-wrapper';
+import Money from '../../money';
+import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+
+const AccumulatorCardBody = ({
+    addToast,
+    connectWithContractUpdate,
+    contract_info,
+    contract_update,
+    currency,
+    current_focus,
+    error_message_alignment,
+    getCardLabels,
+    getContractById,
+    indicative,
+    is_sold,
+    onMouseLeave,
+    removeToast,
+    setCurrentFocus,
+    status,
+    is_positions,
+}) => {
+    const { buy_price, profit, limit_order, sell_price } = contract_info;
+    const { take_profit } = getLimitOrderAmount(contract_update || limit_order);
+    const is_valid_to_sell = isValidToSell(contract_info);
+    const { CURRENT_STAKE, STAKE, TAKE_PROFIT, TOTAL_PROFIT_LOSS } = getCardLabels();
+
+    return (
+        <React.Fragment>
+            <div className='dc-contract-card-items-wrapper'>
+                <ContractCardItem header={STAKE} className='dc-contract-card__stake'>
+                    <Money amount={buy_price} currency={currency} />
+                </ContractCardItem>
+                <ContractCardItem header={CURRENT_STAKE} className='dc-contract-card__current-stake'>
+                    <div
+                        className={classNames({
+                            'dc-contract-card--profit': +profit > 0,
+                            'dc-contract-card--loss': +profit < 0,
+                        })}
+                    >
+                        <Money amount={sell_price || indicative} currency={currency} />
+                    </div>
+                    <div
+                        className={classNames('dc-contract-card__indicative--movement', {
+                            'dc-contract-card__indicative--movement-complete': is_sold,
+                        })}
+                    >
+                        {status === 'profit' && <Icon icon='IcProfit' />}
+                        {status === 'loss' && <Icon icon='IcLoss' />}
```

Comment:
```suggestion
                        {<Icon icon={status === 'profit' ? 'IcProfit' : 'IcLoss' } />}
```

Reply:
unfortunately, status can be also null, that's why in such case it will show the icon for 'lost' 😞